README.md

```diff
@@ -30,7 +30,15 @@ const env = $.env.concat([Integer, NonZeroInteger]);
 The next step is to define a `def` function for the environment:
 
 ```javascript
-const def = $.create(env);
+const def = $.create(true, env);
+```
+
+The first argument to `$.create` determines whether type checking is enabled.
+One may elect to pay the performance cost of run-time type checking only during
+development. For example:
```

Comment:
How about:

*This allows one to only pay the performance cost of run-time type type checking during development.*